.github/workflows/build_pr_documentation.yml

```diff
@@ -63,8 +63,11 @@ jobs:
           pip install .
           cd ..
 
-          cd ${{ inputs.package }}
-          pip install .[dev]
+          if [ "${{ inputs.additional_args }}" != "--not_python_module" ]
```

Comment:
`inputs.additional_args` can potentially hold more args like `--not_python_module --another_arg1 --another_arg2`

would it be possible to check something like:
`if inputs.additional_args includes --not_python_module`

Reply:
Nice idea! Will add that :)